spec/jsSpecs/ares-shell.spec.js

```diff
@@ -123,7 +123,7 @@ describe(aresCmd + ' --run in session', function() {
                     expect(stderr).toContain("ares-shell ERR! [Tips]: This device does not support multiple sessions");
                 }
             } else {
-                expect(stdout.trim()).toBe("hello webOS", stderr);
+                expect(stdout.trim()).toContain("hello webOS", stderr);
```

Comment:
why do you change?
tobe is more strong

Reply:
I added "Set target device to [devivcename]" for this commit.
 expect(stdout.trim()).toBe("hello webOS", stderr) makes always error.